frontend/src/app/App.js

```diff
@@ -38,6 +39,7 @@ class App extends Component {
               <Header />
               <Container>
                 <Route exact path={ROUTES.DASHBOARD} component={Dashboard} />
+                <Route exact path={ROUTES.LEADERBOARDS} component={LeaderboardsPage} />
```

Comment:
Now that I think about it, I'd wrap all the routers to a `Switch` element and have the dashboard as the default route...

Reply:
done [here](https://github.com/salsita/foosball-rating/pull/92/commits/0ca14d1caaa35f00ce4324e704b6cb7f3cf88718#diff-ec0334f02e0502ad4cd3b898702ec2e2)